### PR TITLE
Update polytools.py

### PR DESCRIPTION
**Pull request by wanlinxie**
worked on degree function using recursion, original problem was it was too slow.

### CODE REVIEW

**Comment by debugger22** [comment]
You shouldn't use tabs with spaces in indentation.

File: sympy/polys/polytools.py [hunk 1]
```diff
@@ -4248,34 +4248,40 @@ def _update_args(args, key, value):
...
-    """
-    options.allowed_flags(args, ['gen', 'polys'])
+        """
 
-    try:
-        F, opt = poly_from_expr(f, *gens, **args)
-    except PolificationFailed as exc:
-        raise ComputationFailed('degree', 1, exc)
+	fstr=str(f)
+	h=fstr.split(‘**’)
```
Comment (asmeurer) [inline sympy/polys/polytools.py]: Don't use string manipulation to do symbolics. Also, you are using pretty quotes here.

**Comment by gxyd** [comment]
@wanlinxie can you please address the requested changes here?

**Comment by gxyd** [comment]
Also please use a better PR title. See https://github.com/sympy/sympy/wiki/Development-workflow#writing-pull-request-title-and-description.

**Comment by gxyd** [comment]
Considering the PR description, I think the issue of slowness of degree function was fixed by PR https://github.com/sympy/sympy/pull/12999.

I'm closing this.

**Pull request closed without merging**